docs/src/components/pricing/PricingTable.tsx

```diff
@@ -922,14 +923,30 @@ export default function PricingTable({
   columnHeaderHidden?: boolean;
   plans?: Array<'community' | 'pro' | 'premium'>;
 }) {
+  const router = useRouter();
   const [dataGridCollapsed, setDataGridCollapsed] = React.useState(false);
+
+  React.useEffect(() => {
+    if (router.query['expand-path'] === '*') {
+      setDataGridCollapsed(true);
+    }
+  }, [router.query]);
```

Comment:
So we can share links like https://mui.com/pricing/?expand-path=* that expand the whole pricing page.